shell/platform/darwin/common/framework/Headers/common.h

```diff
@@ -0,0 +1,15 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef FLUTTER_SHELL_PLATFORM_DARWIN_COMMON_FRAMEWORK_HEADERS_COMMON_H_
+#define FLUTTER_SHELL_PLATFORM_DARWIN_COMMON_FRAMEWORK_HEADERS_COMMON_H_
+
+#include <objc/objc.h>
+
+/**
+ * A unique numeric value for a view of an engine.
```

Comment:
Why "of an engine"?

Reply:
I want to express the following two points:
* The IDs are unique within an engine (and are not unique between different engines).
* The views are the engine views, not platform views.

I'm not sure if these points are worth mentioning at all or are there better ways. Do you have a suggestion?

Reply:
That's awesome! Thank you! I'll use this one.